test/adding_tests.md

```diff
@@ -39,10 +39,10 @@ These flags are useful for running against an existing cluster, making use of yo
 [environment setup](/DEVELOPMENT.md#environment-setup).
 
 By importing `github.com/knative/serving/test` you get access to a global variable called
-`test.Flags` which holds the values of [the command line flags](/test/README.md#flags).
+`pkgTest.Flags` which holds the values of [the command line flags](/test/README.md#flags).
```

Comment:
`pkgTest` is the alias you used, it's still called `test`, so for the sake of clarity I wouldn't call it `pkgTest` here.

Reply:
updated